script/Ciel/Day 14/14_01A_CIEL14A_4.txt

```diff
@@ -901,7 +967,9 @@ With a single strike, I sever the thread of life.
 [sha:cd657c3d4254c5b23d835447749735516c2f1ff9]{
 -- Page 152, Offset 38069.
 -- すべての基盤。アルクェイドと繋がっている『星の活力』の供給源である自然を、この一帯だけ殺害する。
-The foundation of everything. I kill the 'vitality of the planet' connected to Arcueid in this region alone; the nature that acts as her source of power.
+I kill the foundation of everything.
+ I kill all of the nature in this region,
+ the source that supplied Arcueid with the 'vitality of the planet'.
```

Comment:
すべての基盤 is liable to be misintepreted with this TL, so...
>I kill the underlying bedrock of everything.

Since it's the planet we're talking about here.
>...with the 'planet's force of life'.